packages/composites/proof-of-reserves/src/endpoint/reserves.ts

```diff
@@ -38,19 +38,30 @@ const inputParameters: InputParameters = {
 
 export const execute: ExecuteWithConfig<Config> = async (input, context, config) => {
   const validator = new Validator(input, inputParameters, paramOptions)
-
   const jobRunID = validator.validated.jobRunID
   const protocol = validator.validated.data.protocol.toUpperCase()
   const indexer: Indexer = validator.validated.data.indexer.toUpperCase()
   const confirmations = validator.validated.data.confirmations
 
   const protocolOutput = await runProtocolAdapter(jobRunID, context, protocol, input.data, config)
+  const validatedInput = { ...protocolOutput }
+  if (
+    validator.validated.data.disableAddressValidation &&
+    validator.validated.data.disableAddressValidation !== 'false'
+  )
+    validatedInput.result = validateAddresses(indexer, validatedInput.result)
+  if (
+    validator.validated.data.disableDuplicateAddressFiltering &&
+    validator.validated.data.disableDuplicateAddressFiltering !== 'false'
+  )
+    validatedInput.result = filterDuplicates(validatedInput.result)
```

Comment:
Yeah I see the issue.  Both methods technically work if the value is only ever set to "true" or "false", but in the event of a mistype, it would default to true. (ex: "fase" would be set to true and not false).  I committed your suggestion to fix this.